src/battle_script_commands.c

```diff
@@ -4232,13 +4232,19 @@ static u32 GetMonHoldEffect(struct Pokemon *mon)
     return holdEffect;
 }
 
+bool32 ShouldCombineLevelUpMessages(void)
+{
+    return (B_LEVEL_UP_NOTIFICATION >= GEN_9);
+}
```

Comment:
I think this is a bit to much. We don't write functions for gen config comparison and if we started it would explode.

Reply:
https://github.com/PokemonSanFran/pokeemerald-expansion/commit/4d92c7d490b4c31a4b4881f52350a1ec717da809